website/content/docs/platform/k8s/vso/examples.mdx

```diff
@@ -11,6 +11,34 @@ The Operator project provides the following examples:
 - Sample use-cases are documented [here](https://github.com/hashicorp/vault-secrets-operator#samples)
 - A Terraform based demo can be found [here](https://github.com/hashicorp/vault-secrets-operator/tree/main/demo)
 
+## Configuring the JWT auth method to avoid API calls to the Kubernetes API Server
+
+Vault Secrets Operator supports using the [JWT auth method](/vault/docs/platform/k8s/vso/api-reference#vaultauthconfigjwt).
+In the default JWT configuration this involves an API call from the Vault server back to the Kubernetes API server to [verify](/vault/docs/auth/jwt#jwt-verification) the JWT.
+It is possible to configure Vault to use [`jwt_validation_pubkeys`](/vault/api-docs/auth/jwt#jwt_validation_pubkeys) fields of the JWT auth configuration
+to work in the Kubernetes environment and avoid the API call back to Kubernetes, improving performance of login requests.
+
+### Configure the JWT Auth method in Vault to use the JWKS keys from Kubernetes:
+
+* Fetch the JWKS key from Kubernetes using `kubectl`
+
+```shell
+demo $ kubectl get --raw "$(kubectl get --raw /.well-known/openid-configuration | jq -r '.jwks_uri' | sed -r 's/.*\.[^/]+(.*)/\1/')" | jq  '.keys | .[0]'
+{
+  "use": "sig",
+  "kty": "RSA",
+  "kid": "o2Tjqi7n9ye3VW6L8....",
+  "alg": "RS256",
+  "n": "2SixHcuzdYrj.....",
+  "e": "AQAB"
+}
+````
+
+* Copy/paste this JSON object into your favourite JWKS->PEM convertor and use this PEM file as the
+PEM file in your `jwt_validation_pubkeys` field when [configuring](/vault/api-docs/auth/jwt#configure) the JWT Auth Method in Vault.
+
+Should the JWT Signing Key used by Kubernetes be rotated, the process should be repeated with the new Key.
```

Comment:
It would be good if we could expand a bit more this, and maybe add some extra emphasis. Maybe move it into a Note like here: https://github.com/hashicorp/vault/blob/ff4489b5af9eb594ddd86301e1f6dab0869a41cd/website/content/docs/platform/k8s/vso/installation.mdx#L57